src/gui/newwizard/SetupWizardWindow.cpp

```diff
@@ -0,0 +1,152 @@
+#include "SetupWizardWindow.h"
+
+#include <QLabel>
+
+#include "Pagination.h"
+#include "ui_SetupWizardWindow.h"
+
+namespace OCC::Wizard {
+
+SetupWizardWindow::SetupWizardWindow(QWidget *parent)
+    : QDialog(parent)
+    , _ui(new ::Ui::SetupWizardWindow)
+    //    , _transitionLayout(new QHBoxLayout)
+    , _transitionProgressIndicator(new QProgressIndicator)
+{
+    _ui->setupUi(this);
+
+    // make sure to initialize these after UI has been set up
+    _pagination.reset(new Pagination(_ui->paginationLayout));
+
+    slotHideErrorMessageWidget();
+
+    _contentLayout.reset(new QStackedLayout(_ui->contentWidget));
```

Comment:
Can we use the .ui?

Mixing manual layouting and a .ui usually doesn't end well